### DISCUSSION TITLE
[Question]: prompt中英文混用会影响模型性能吗？

### DISCUSSION THREAD

**Discussion by YAETHeryk** [QwenLM/Qwen3#979] [Q&A]
### Has this been raised before?

- [X] I have checked [the GitHub README](https://github.com/QwenLM/Qwen2.5).
- [X] I have checked [the Qwen documentation](https://qwen.readthedocs.io) and cannot find an answer there.
- [X] I have searched [the issues](https://github.com/QwenLM/Qwen2.5/issues?q=is%3Aissue) and there is not a similar one.
- [X] I confirm that this is not a bug report, a feature request, or a badcase.

### Description

langchain里有一些组件自带英语的prompt，但是构建的应用都是中文对话，是否有必要手动重写langchain的一些函数，统一语言。

**Comment by jklj077** [comment]
Hi, it should work more or less. After all, our default system prompt is in English, so mixed languages is common usecase. But I don't think you can achieve the best results with that kind of prompts. My advice is to first try the default to see if the prototype works and tune the prompts later.
